src/chips/nrf51822/gpio.rs

```diff
@@ -55,19 +127,54 @@ impl hil::gpio::GPIOPin for GPIOPin {
     }
 
     fn toggle(&self) {
-        unimplemented!();
+        // TODO: check need for a atomic XOR operator
```

Comment:
Right, because of the execution model, I don't see a reason for an atomic operation.